src/test/java/org/opentripplanner/transit/raptor/speed_test/SpeedTest.java

```diff
@@ -57,17 +58,24 @@ public class SpeedTest {
   private SpeedTest(SpeedTestCmdLineOpts opts) {
     this.opts = opts;
     this.config = SpeedTestConfig.config(opts.rootDir());
-    OtpModel otpModel = loadGraph(opts.rootDir(), config.graph);
-    this.graph = otpModel.graph;
-    this.transitModel = otpModel.transitModel;
+    TestOtpModel model = loadGraph(opts.rootDir(), config.graph);
+    this.graph = model.graph();
+    this.transitModel = model.transitModel();
 
     this.tcIO = new CsvFileIO(opts.rootDir(), TRAVEL_SEARCH_FILENAME);
 
     // Read Test-case definitions and expected results from file
     this.testCaseInputs = filterTestCases(opts, tcIO.readTestCasesFromFile());
 
-    this.router = new Router(graph, transitModel, RouterConfig.DEFAULT, timer.getRegistry());
-    this.router.startup();
+    this.serverContext =
+      new DefaultServerContext(
+        graph,
+        transitModel,
+        RouterConfig.DEFAULT,
+        timer.getRegistry(),
+        false
+      );
+    // TODO serverContext.startup();
```

Comment:
I did not understand why it was necessary to call `startup()`, so I commented it out to see if the tests would fail, then I fixed the test and removed most of them, but forgot this + one other. Thanks for spotting this.